### ISSUE TITLE
mascots unable to use local picture links

### ISSUE THREAD

**Issue by yarrmateys** [zixaphir/appchan-x#945]
considering the custom made ones are likely going to remain on the user's pc, would it be possible to make the script use a local link like file:///D:/mascots/mascot.png or similar, or is this a browser limitation of some sort that those don't work?

**Comment by zixaphir** [comment]
I'm not exactly sure why local file URIs don't work.

**Comment by zixaphir** [comment]
Yeah, browser thing: http://www.websina.com/bugzero/kb/browser-file-url.html

**Comment by anonymau5** [comment]
Perhaps a home server could host the images, if you really want to keep the traffic local.
I put all mine up on imgur and don't worry about it.

**Comment by yarrmateys** [comment]
yeah, i mostly wanted to use that for testing, but i guess i can use dropbox for that.

**Comment by ccd0** [comment]
You could also use data URIs.
http://bran.name/dump/data-uri-generator/
